crates/ursa-network/src/service.rs

```diff
@@ -218,6 +220,8 @@ where
     cached_content: CacheSummary,
     /// Content summaries from other nodes.
     peer_cached_content: HashMap<PeerId, CacheSummary>,
+    /// Interval for random Kademlia walks
```

Comment:
```suggestion
    /// Interval for random Kademlia walks.
```

Reply:
Don't worry about this one. I'm planning to do some clean up with the comments soon.